### PR TITLE
Merge release 1.14 to master

### PR DESCRIPTION
**Pull request by alanprot**
Merge back release branch

### CODE REVIEW

File: VERSION [hunk 1]
```diff
@@ -1 +1 @@
-1.13.0
+1.14.0
```
Comment (yeya24) [inline VERSION:1]: When merging to master, should `master` branch to be `1.14.0` or `1.15.0-dev`?

Comment (alanprot) [reply VERSION:1]: It was 1.13 before this release.

Comment (alanprot) [reply VERSION:1]: https://github.com/cortexproject/cortex/pull/4793/files

Comment (yeya24) [reply VERSION:1]: Let's merge on green.

**Review by yeya24 — APPROVED** [review]

**Pull request merged**